src/helm/benchmark/scenarios/gpqa_scenario.py

```diff
@@ -63,8 +63,15 @@ def get_instances(self, output_path: str) -> List[Instance]:
                 Reference(Output(text=row["Incorrect Answer 3"].strip()), tags=[]),
             ]
             random.shuffle(references)
-            split = TRAIN_SPLIT if idx in EXCLUDED_TRAIN_EXAMPLES[self.subset] else TEST_SPLIT
-            instance = Instance(input=input, references=references, split=split)
+            if idx in TRAIN_EXAMPLE_INDICES[self.subset]:
+                extra_data = {
+                    "chain_of_thought": row["Explanation"],
+                    "chain_of_thought_prefix": "Let's think step by step: ",
+                    "chain_of_thought_suffix": "The correct answer is ",
```

Comment:
Remove this; you can just put this in the `AdapterSpec` later.

Reply:
Sounds good, if we can put those in `AdapterSpec` it makes it more straightforward too.